core/budfox/budfox.js

```diff
@@ -33,46 +33,46 @@ var BudFox = function(config) {
   // relay a marketUpdate event
   this.marketDataProvider.on(
     'marketUpdate',
-    e => this.emit('marketUpdate', e)
+    e => this.emit('marketUpdate', e),
```

Comment:
What do you think this improves over the old style?

Reply:
All the styling is conform one basic style guideline. If the code should have trailing comma's after the last statement is a different discussion.

I could add a PR to use ESLint. The project will be better of if all the code conforms to one style.

Reply:
I don't think the current project uses trailing commas anywhere else, would you be okay with conforming to a style with no trailing commas?

Reply:
I suggest using `eslint-config-airbnb-base,` `eslint-plugin-import`, `eslint-plugin-vue` and the recommended `ESLint` rules. 

This would enforce the code to be conform the `AirBnB` style guideline as follows:

```
Semicolon: Required
Trailing Commas: Required
Template strings: Prefer
Import Extensions: None
Space before function parentheses: None
Object Curly Spacing: Yes
Array Bracket Spacing: None
Underscored functions: None
Object Destructuring: Prefer
React Ordering: Opinionated
React Prop Validation: Required
Arrow Functions Return Assignment: No
Object Property Shorthand: Prefer
```